vpr/src/place/net_cost_handler.cpp

```diff
@@ -1765,33 +1470,29 @@ static double get_net_cost(ClusterNetId net_id, const t_bb& bb) {
      * channel capacity.   Do this for x, then y direction and add.  */
 
     double ncost;
-    ncost = (bb.xmax - bb.xmin + 1) * crossing * chanx_place_cost_fac[bb.ymax][bb.ymin - 1];
-    ncost += (bb.ymax - bb.ymin + 1) * crossing * chany_place_cost_fac[bb.xmax][bb.xmin - 1];
+    ncost = (bb.xmax - bb.xmin + 1) * crossing * chanx_place_cost_fac_[bb.ymax][bb.ymin - 1];
+    ncost += (bb.ymax - bb.ymin + 1) * crossing * chany_place_cost_fac_[bb.xmax][bb.xmin - 1];
 
-    return (ncost);
+    return ncost;
```

Comment:
We should have a 3rd term: 
ncost += (layer_max-layer_min+1) * chanz_place_cost_fac

Big TODO!